tests/test_main.py

```diff
@@ -1,6 +1,260 @@
-# TODO: add tests
-from app.main import hello_world
+import datetime
 
+import pytest
 
-def test_hello_world():
-    assert hello_world() == "Hello, world!"
+from app.cafe import Cafe
+from app.errors import NotVaccinatedError, VaccineError, OutdatedVaccineError, NotWearingMaskError
+from app.main import go_to_cafe
+
+
+def test_errors_hierarchy():
+    assert NotVaccinatedError.__bases__ == (VaccineError, ), (
+        "NotVaccinatedError should inherit only VaccineError class"
+    )
+    assert OutdatedVaccineError.__bases__ == (VaccineError, ), (
+        "OutdatedVaccineError should inherit only VaccineError class"
+    )
+
```

Comment:
I'd suppose adding here assertion about NotWearingMaskError should not be inherited from VaccineError
```assert VaccineError not in NotWearingMaskError.__bases__```

Reply:
Done